src/ResourceManager/Resources/Commands.Resources/help/Get-AzureRmManagementPartner.md

```diff
@@ -0,0 +1,92 @@
+---
+external help file: Microsoft.Azure.Commands.Resources.dll-Help.xml
+Module Name: AzureRM.Resources
+online version:
+schema: 2.0.0
+---
+
+# Get-AzureRmManagementPartner
+
+## SYNOPSIS
+Get the current management partner id
+
+## SYNTAX
+
+```
+Get-AzureRmManagementPartner [[-PartnerId] <String>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Get management partner id for the current user under this tenant
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzureRmManagementPartner
```

Comment:
For all examples in markdown files, please give the output of each cmdlet call so users know what to expect.